source/blog/2016-04-20-scala-210-and-java7.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: Final Scala 2.10 and Java 7 releases
+published: true
+post_author:
+  display_name: Kevin Oliver
+  twitter: kevino
+tags: Finagle, Finatra, Util, Scrooge, TwitterServer, Ostrich
+---
+
+The upcoming releases of our family of projects will be the final release for
+Scala 2.10 and Java 7. These should be:
+
+* Finagle 6.35.0
+* Finatra 2.2.0
```

Comment:
i'll change it, but i'm surprised you're not gonna bump minor even with all the dependency changes.